grafana/scylla-dash-per-server.master.json

```diff
@@ -4078,7 +4078,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "ops",
```

Comment:
This is right, but we should also fix the long standing bug in the query definition, which should take irate() (time derivative) of the metric

Reply:
@tgrabiec can you please open an issue for this one?

Reply:
There is already an issue for this: https://github.com/scylladb/scylla-grafana-monitoring/issues/158